packages/app/src/app/pages/Sandbox/Editor/Workspace/Project/index.tsx

```diff
@@ -0,0 +1,192 @@
+import * as React from 'react';
+import { observer } from 'mobx-react-lite';
+import { Link } from 'react-router-dom';
+import {
+  sandboxUrl,
+  githubRepoUrl,
+  profileUrl,
+} from '@codesandbox/common/lib/utils/url-generator';
+
+import TeamIcon from 'react-icons/lib/md/people';
+import { getSandboxName } from '@codesandbox/common/lib/utils/get-sandbox-name';
+import { UserWithAvatar } from '@codesandbox/common/lib/components/UserWithAvatar';
+import Stats from 'app/pages/common/Stats';
+import PrivacyStatus from 'app/components/PrivacyStatus';
+import GithubBadge from '@codesandbox/common/lib/components/GithubBadge';
+import EditableTags from 'app/components/EditableTags';
+import Tags from '@codesandbox/common/lib/components/Tags';
+import Tooltip from '@codesandbox/common/lib/components/Tooltip';
+
+import getTemplateDefinition from '@codesandbox/common/lib/templates';
+
+import { useSignals, useStore } from 'app/store';
+import {
+  Item,
+  UserLink,
+  StatsContainer,
+  PrivacyContainer,
+  PropertyValue,
+  PropertyName,
+} from './elements';
+import TitleComponent from './Title';
+import FrozenComponent from './Frozen';
+import DescriptionComponent from './Description';
+import TemplateComponent from './Template';
+// import AliasComponent from './Alias';
+
+const Project = ({ editable }: { editable: boolean }) => {
+  const { editor, workspace } = useStore();
+  const { notificationAdded, workspace: workspaceSignals } = useSignals();
+
+  const changeTags = (newTags: string[], removedTags: string[]) => {
+    const { tags } = editor.currentSandbox;
+
+    if (tags.length > 5) {
+      notificationAdded('You can have a maximum of 5 tags', 'error');
+      return;
+    }
+
+    const tagRemoved = newTags.length < tags.length && removedTags.length === 1;
+    if (tagRemoved) {
+      removedTags.forEach(tag => {
+        workspaceSignals.tagRemoved({ tag });
```

Comment:
@SaraVieira Use destuctured function name (see above)

```suggestion
        tagRemoved({ tag });
```